apps/gallery/js/startup.js

```diff
@@ -273,11 +273,17 @@
       window.performance.mark('contentInteractive');
 
       // Remove the attribute to trigger gaia header font-fit logic, defer it
-      // til now to postpone the overhead.
-      var headers = document.querySelectorAll('gaia-header');
-      for (var i = 0; i < headers.length; ++i) {
-        headers[i].removeAttribute('no-font-fit');
-      }
+      // til now to postpone the overhead. Add setTimeout to lazyload
+      // gaia-header.js aynchronously.
+      setTimeout(function() {
```

Comment:
LazyLoader.load() is already async, so do you really need setTimeout() here?  If it works without adding the setTimeout() I think that would be better.

If you tap the select button at the bottom right as soon as the app launches can you get to that screen before the header loads or before it is localized?  Maybe the event handlers for the thumbnails and the select button shouldn't be registered until the header is lazy loaded. If so, then we should do performance.mark('contentInteractive') when the event handlers are registered after the lazy load.

I'm not sure that it matters in practice, but that seems like the right way to do it.